cypress/plugins/index.js

```diff
@@ -20,7 +20,7 @@ const cucumber = require("cypress-cucumber-preprocessor").default;
  * @type {Cypress.PluginConfig}
  */
 // eslint-disable-next-line no-unused-vars
-module.exports = (on, config) => {
+module.exports = (on, _config) => {
```

Comment:
there's an eslint comment to ignore this so shouldn't be necessary. if you decide to keep _config, then remove the comment and vice versa.